cmd/controller/controller.go

```diff
@@ -24,8 +24,11 @@ var (
 	configPath    string
 	masterURL     string
 	importerImage string
-	pullPolicy    string
-	verbose       string
```

Comment:
I think this might be cleaner if we share verbosity (and maybe even pull policy) between both controllers

Reply:
Done